Library/Homebrew/migrator.rb

```diff
@@ -42,6 +42,8 @@ def initialize(formula, tap)
         #{formula.name} from #{formula.tap} is given, but old name #{formula.oldname} was installed from #{tap}
 
         Please try to use #{name} to refer the formula
+
+        To force migrate use `brew migrate --force #{formula.oldname}`
```

Comment:
The output may need some improvement. It looks strange to have these empty lines. Maybe it's a good idea to eliminate the if-else branch as well.

In the mean time, the manpage need update as well.